src/api_routes_websocket.rs

```diff
@@ -117,140 +131,202 @@ use std::{
   ops::Deref,
   result,
   str::FromStr,
-  sync::{Arc, Mutex},
   time::{Duration, Instant},
 };
-use tracing::{debug, error, info};
+use tracing::{debug, error};
 
-/// Entry point for our route
-pub async fn websocket(
-  req: HttpRequest,
-  body: web::Payload,
-  context: web::Data<LemmyContext>,
-  rate_limiter: web::Data<RateLimitCell>,
+/// How often heartbeat pings are sent
+const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
This seems way too much, especially for mobile. Isnt 30 or 60 seconds enough?

Reply:
Sure, I'll increase to 30 and 60s, from here: https://stackoverflow.com/questions/12815231/controlling-the-heartbeat-timeout-from-the-client-in-socket-io

Reply:
Not sure where you see 30 and 60, the first answer says `heartbeat interval = 25s and heartbeat timeout = 60s`.